core/src/main/java/org/bitcoinj/uri/BitcoinURI.java

```diff
@@ -118,7 +117,7 @@ public static BitcoinURI of(String uri) throws BitcoinURIParseException {
      * @param network The network the URI is from
      * @throws BitcoinURIParseException If the input fails Bitcoin URI syntax and semantic checks.
      */
-    public static BitcoinURI of(String uri, @Nonnull Network network) throws BitcoinURIParseException {
+    public static BitcoinURI of(String uri, Network network) throws BitcoinURIParseException {
```

Comment:
This should stay `@Nonnull`, I think. Rather than pass `null` the other constructor should be used.

Reply:
But, maybe you're removing the `@Nonnull` because it is our default?

Reply:
Yes, but I'll add it back because it's out of scope of this PR.

However I've forgotten a `Objects.requireNotNull()` on that parameter which I'll add before merging.